cordova-lib/src/plugman/platforms/ios.js

```diff
@@ -184,13 +192,65 @@ module.exports = {
 
 
         var xcBuildConfiguration = xcodeproj.pbxXCBuildConfigurationSection();
-        var plist_file_entry = _.find(xcBuildConfiguration, function (entry) { return entry.buildSettings && entry.buildSettings.INFOPLIST_FILE; });
-        var plist_file = path.join(project_dir, plist_file_entry.buildSettings.INFOPLIST_FILE.replace(/^"(.*)"$/g, '$1').replace(/\\&/g, '&'));
-        var config_file = path.join(path.dirname(plist_file), 'config.xml');
 
-        if (!fs.existsSync(plist_file) || !fs.existsSync(config_file)) {
+        // CB-9033
+        var plist_file_index;
+        var plist_file_entry = _.find(xcBuildConfiguration, function (entry,index) { 
+            if (entry.buildSettings && entry.buildSettings.INFOPLIST_FILE && entry.buildSettings.SKIP_INSTALL != 'YES') { 
```

Comment:
ok, I will change it, even though I don't believe it would make a difference... unless there are cases I've missed.
if it is part of the cordova-lib coding requirements, it would be good to update the jslint rules accordingly so it is catched during the test phase